### PR TITLE
Import declarations expanded view styles updated

### PR DESCRIPTION
**Pull request by roonyh**

### CODE REVIEW

File: modules/web/js/ballerina/components/import-declaration-expanded.jsx [hunk 1]
```diff
@@ -17,15 +17,21 @@
...
         const importDeclarationHeight = 30;
-        const importDeclarationWidth = 200;
+        const importInputHeight = 40;
+        const importDeclarationWidth = 300;
         const leftPadding = 10;
+        const topGutter = 10;
+        const iconSize = 20;
         const importElements = [];
-        let lastImportElementY = bBox.y;
+        let lastImportElementY = bBox.y + topGutter;
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/components/import-declaration-expanded.jsx]: Shall we unify all these constants (importInputHeight, importDeclarationWidth, etc)  values in to the consts file

File: modules/web/js/ballerina/components/import-declaration-expanded.jsx [hunk 2]
```diff
@@ -39,10 +45,33 @@ export default class importDeclarationExpanded extends React.Component {
             lastImportElementY += importDeclarationHeight;
         });
 
+        const textBoxBBox = {
+            x: bBox.x + 5,
```
Comment (sumuditha-viraj) [inline modules/web/js/ballerina/components/import-declaration-expanded.jsx]: Can we remove magic numbers introducing proper variable names?

File: modules/web/js/ballerina/components/import-declaration-expanded.jsx [hunk 3]
```diff
@@ -39,10 +45,33 @@ export default class importDeclarationExpanded extends React.Component {
...
+
+        this.context.renderer.renderTextBox(options);
+
         return (
             <g className="package-definitions-collection">
                 {importElements}
+                <rect x={ bBox.x } y={ lastImportElementY } height={importInputHeight} width={importDeclarationWidth} />
+                <image width={ iconSize } height={ iconSize } className="property-pane-action-button-delete"
+                    onClick={this.props.onCollapse} xlinkHref={ ImageUtil.getSVGIconString('hide') }
+                    x={bBox.x + importDeclarationWidth - iconSize - 6 } y={bBox.y + topGutter + 3}/>
```
Comment (sumuditha-viraj) [inline modules/web/js/ballerina/components/import-declaration-expanded.jsx]: Remove magic numbers.

File: modules/web/js/ballerina/components/import-declaration.jsx [hunk 4]
```diff
@@ -33,7 +35,7 @@ export default class importDeclaration extends React.Component {
         number.y = text.y;
         return (
             <g className="package-definition-head" onClick={ e => {this.props.onClick(e);} }>
-                <rect x={ bBox.x } y={ bBox.y } width={110} height={ headerHeight } rx="0" ry="0" className="package-definition-header"/>
+                <rect x={ bBox.x } y={ bBox.y } width={136} height={ headerHeight } rx="0" ry="0" className="package-definition-header"/>
```
Comment (sumuditha-viraj) [inline modules/web/js/ballerina/components/import-declaration.jsx]: Remove magic numbers

**Pull request merged**